Automattic/syntaxhighlighter
# Fix vulnerabilities

### Overview

Fix the vulnerabilities and update the `cgb-scripts` package to the last version.

### Testing

1. Run the build with the current version and save the file content for comparison.
2. Run the build in this branch.
3. Compare both. It should be the same.

Also, to confirm if everything continues working properly (copied from #118):

4. I installed the plugin in a local test site.
5. I created a post with a Syntax Highlighter block and tested it in the frontend.
6. I navigated to the admin settings page to ensure that there are no errors.
